source/index.html.md

```diff
@@ -333,64 +333,71 @@ Depending on card scheme and merchant category, the name on the card might be
 necessary for approval of credits. It may be provided through the optional
 parameter `card[name]`.
 
-## Recurring payments
+## Series of transactions
 
-Recurring payments enable you to repeatedly charge cardholders without providing
-CSC or other means of cardholder authentication for subsequent payments.
+Clearhaus currently support only one type of subscription billing:
```

Comment:
```suggestion
Clearhaus currently supports only one type of subscription billing:
```

Reply:
https://erinwrightwriting.com/singular-or-plural-pronouns-for-a-company/

> British English generally treats collective nouns as plural rather than singular. So, if you follow British English instead of American English, you can usually use the plural third-person pronouns *they*, *them*, *their*, and *themselves* for an individual company.

However, I believe we use American English in our code and documentation (e.g. authorization, not authorisation).

Reply:
And let me add "argh, wtf, why are there so significant differences between British and American English!?" :cry: